source/documentation/runbooks/010-disaster-recovery-plan.html.md.erb

```diff
@@ -0,0 +1,189 @@
+
+## Overview
+This runbook provides step-by-step guidance for recovering the 'Find MoJ Data' Django application hosted on the Ministry of Justice's internal Cloud Platform (CP) using AWS EKS.
+It covers the actions required by the Data Catalogue team in the event of a disaster, focusing on minimizing downtime and data loss.
+
+## Summary
+- Find MoJ Data is a Django application that calls a backend service [DataHub] for all of its data. Both services are hosted on the MoJ's [Cloud Platform] (CP), and use CP for integrated services like the application database ([AWS RDS PostgreSQL]) and search service (AWS OpenSearch).
+
+- Both Find MoJ Data and DataHub deployments have RDS PostgreSQL databases deployed via CP and so get [automated daily backups] and the last 7 are retained.
+
+  > **Note**: due to an implementation bug, as of 2024-09-03 the snapshot backups are taken _**every other morning**_, meaning the oldest backup is 12-13 days old, and that the most recent backup can be up to 48 hours old.
+
+- Following loss or corruption of database data, we can either re-ingest metadata through GitHub Actions workflows, or restore from one of these backups by [following CP's RDS snapshot restore guidance], depending on the scenario.
+
+  - The process for restoring from snapshot should be tested every 6-12 months to ensure the team is familiar with the process, and that there are no unexpected issues.
+
+  - The Find MoJ Data RDS database contains:
+    - app usage data, which would not be overly detrimental to operation of the service if it were to be lost
+    - survey data, which should be retained where possible
+
+  - The DataHub RDS database contains catalogue metadata aggregated from other data-handling services along with application data (user data, usage data). _(As of 2024-09-03)_ The current aims for our DataHub are to aggregate metadata as-is from data sources, so total loss of this metadata would not result in loss of information for the business, as it could all be re-ingested into the catalogue application.
+
+- The Find MoJ Data service is not considered to be a critical service, and as such we have single RDS instances for each of our deployed environments. We do not have multi-Availability Zone (AZ) deployments and/or read replicas for our databases, as the business will be able to tolerate reasonable outages due to aws-level disruptions (e.g. AZ- or region-level outages).
+
+- In the medium term, we do not expect more than 50 simultaneous users for any length of time. At full operation, we could expect to see hundred-to-thousands of unique monthly users, and at this time we may need to re-visit our recovery objectives.
+
+## Scope
+- **Application**: Django-based Python application
+- **Backend services**: DataHub
+- **Hosting**: MoJ's Cloud Platform: AWS EKS
+- **Database**: AWS RDS PostgreSQL
+- **Key Components**:
+  - Django application code
+  - DataHub application
+  - [AWS RDS PostgreSQL database]
```

Comment:
what do the square brackets mean here

Reply:
Those are markdown links. The link text is here, and the links are at the bottom of the document. Makes text easier to parse especially when there are links in the middle of paragraphs.